src/app/public/modules/dropdown/dropdown-menu.component.ts

```diff
@@ -365,13 +365,6 @@ export class SkyDropdownMenuComponent implements AfterContentInit, OnDestroy {
             }
             this.sendMessage(SkyDropdownMessageType.FocusTriggerButton);
             break;
-
-          case ' ': // Spacebar.
-          case 'enter':
```

Comment:
I don't know why I had these in here to begin with. The `click` event handles the pointer event, as well as the `ENTER` and `SPACE` key automatically. Adjusted the unit test fixture to accommodate this change.

Reply:
I'm guessing b/c this used to not be a button? It was just a div and browsers didn't handle enter/space for a div.